apps/01PigLatin.js

```diff
@@ -10,7 +10,39 @@ var rl = readline.createInterface({
 
 function pigLatin(word) {
 
-  // Your code here
+  var firstLetter = word[0];
+
+  word = word.toLowerCase();
+
+  var vowelIndex = -1;
+
+  if ( ( word.indexOf('a') > -1 && word.indexOf('a') < vowelIndex ) || vowelIndex === -1 ) {
+    vowelIndex = word.indexOf('a');
+  }
+  if ( ( word.indexOf('e') > -1 && word.indexOf('e') < vowelIndex ) || vowelIndex === -1 ) {
+    vowelIndex = word.indexOf('e');
+  }
+  if ( ( word.indexOf('i') > -1 && word.indexOf('i') < vowelIndex ) || vowelIndex === -1 ) {
+    vowelIndex = word.indexOf('i');
+  }
+  if ( ( word.indexOf('o') > -1 && word.indexOf('o') < vowelIndex ) || vowelIndex === -1 ) {
+    vowelIndex = word.indexOf('o');
+  }
+  if ( ( word.indexOf('u') > -1 && word.indexOf('u') < vowelIndex ) || vowelIndex === -1 ) {
+    vowelIndex = word.indexOf('u');
+  }
+  if ( ( word.indexOf('y') > -1 && word.indexOf('y') < vowelIndex ) || vowelIndex === -1 ) {
+    vowelIndex = word.indexOf('y');
+  }
+
+  var firstPart = word.slice(0, vowelIndex);
```

Comment:
I recommend putting these two lines above the `return` statement in the `else` block. That way they only run if the `if` statement is false (currently they always run).